CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.0]
+### Uncategorized
+- Export types; Move NonceTracker to named export instead of default ([#53](https://github.com/MetaMask/nonce-tracker/pull/53))
+- BREAKING: Drop support for Node.js v14 ([#54](https://github.com/MetaMask/nonce-tracker/pull/54))
```

Comment:
Should we mention what the new minimum Node version is now? Also, what do you think about renaming the change category and putting BREAKING at the top?

``` suggestion
### Added
- Add ancillary types ([#53](https://github.com/MetaMask/nonce-tracker/pull/53))

### Changed
- **BREAKING:** Drop support for Node.js v14; minimum required version is now v16 ([#54](https://github.com/MetaMask/nonce-tracker/pull/54))
- Move NonceTracker to named export instead of default ([#53](https://github.com/MetaMask/nonce-tracker/pull/53))